tsconfig.base.json

```diff
@@ -7,7 +7,7 @@
     "strict": true,
     "resolveJsonModule": true,
     "esModuleInterop": true,
-    "moduleResolution": "node",
+    "moduleResolution": "node16",
```

Comment:
why is it required to update `moduleResolution`?

Reply:
I got the following on the `createViemAccount` import using `node`. checking back an additional tsconfig was required as a result (see f49f51d4cf3046fab7aca8813a5f493531df9d2c)
```
Cannot find module '@privy-io/server-auth/viem' or its corresponding type declarations.
  There are types at '/Users/adamfuller/Documents/GitHub/agentkit/node_modules/@privy-io/server-auth/dist/viem.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.
```